features/facebookImport/test/mocks/zipfile-mock.js

```diff
@@ -17,6 +17,18 @@ export class ZipFileMock {
         return this._entries[entry];
     }
 
+    async stat(entry) {
+        const entryContent = this._entries[entry];
+        return {
+            getId: () => entry,
+            getTime: () => "",
+            getName: () => entry.substr(this.id.length),
+            getSize: () => entryContent.length,
```

Comment:
Is this why you're using `getSize` above?

Reply:
The actual name of the API call is `getSize` - that's what _should_ be used, how I see it. Accessing the `size` property is accessing (what's supposed to be, but physically can't be) private state.

Reply:
The TypeScript code of the browser pod interface aspects the result of stats to be something of type `Stats`. In Kotlin the returned value is a property `size`. So looks like an inconsistencies between two interfaces.

Reply:
Hm, should still work, since `RemoteClientPod` does have a `getSize()` method which just returns the `size` property. Anyway, not important enough to block this I think, hence the TODO.